src/main/java/aspects/UserRestoreAspect.java

```diff
@@ -98,9 +99,7 @@ public void closeRestoreFactory(JoinPoint joinPoint) throws Throwable {
 
     private HqAuth getAuthHeaders(String domain, String username, String sessionToken) {
         HqAuth auth;
-        if (sessionToken != null && sessionToken.equals(authKey)) {
-            auth = new BasicAuth(touchformsUsername, touchformsPassword);
```

Comment:
@wpride tests are failing, and i think it's because the touchforms user auth has been removed here without implementing mac digest auth for the restore as user endpoint